snowshu/core/replica/replica_factory.py

```diff
@@ -72,15 +71,15 @@ def _execute(self,
                     flags=re.IGNORECASE,
                 )
             )
-            logger.info(f"Incremental target catalog: {incremental_target_catalog}")
```

Comment:
Was this log removed intentionally? If it was noisy, we should be able to decrease log level to debug, right?

Reply:
I think I must have merged this log in some previous PR as it was during development process, as it's not present on master branch. I think it is not really necessary to log it, hence I dropped it